test/e2e/lib/framework/pageSetups.ts

```diff
@@ -31,16 +31,20 @@ export const DEFAULT_SETUPS = [
 export function asyncSetup(options: SetupOptions) {
   let body = options.body || ''
 
+  function formatSnippet(url: string, globalName: string) {
+    return `(function(r,a,w,l,s) {
```

Comment:
"rawls" is a [The Wire reference](https://en.wikipedia.org/wiki/William_Rawls)

Reply:
nice one but I don't think that the wire references are publicly shared 🙂 
what about something dog related instead? `hound` would fit 😉

Reply:
Ok for `hound` :)